anecdote/assets/logs/anecdote.yaml

```diff
@@ -0,0 +1,55 @@
+id: anecdote
+metric_id: anecdote
+backend_only: true
+pipeline:
+  type: pipeline
+  name: Anecdote Log Pipeline
+  enabled: true
+  filter:
+    query: "source:anecdote"
+  processors:
+    - type: service-remapper
+      name: Define `service` as the official service of the log
+      enabled: true
+      sources:
+        - service
+    - type: date-remapper
+      name: Define `content.timestamp` as the official date of the log
+      enabled: true
+      sources:
+        - content.timestamp
+    - type: attribute-remapper
+      name: Map `id` to `log_id`
+      enabled: true
+      sources:
+        - id
+      sourceType: attribute
+      target: log_id
+      targetType: attribute
+      preserveSource: true
```

Comment:
Can you set all `preserveSource: true` to false, it will duplicate the value, and I believe this is not needed here